### PR TITLE
Move PeakRDL-Markdown to SystemRDL namespace

### PR DESCRIPTION
**Pull request by MarekPikula**
Connected to SystemRDL/PeakRDL-Markdown#2

### CODE REVIEW

**Review by amykyta3 — APPROVED** [review]

**Comment by MarekPikula** [comment]
Thanks for the approval. I'm afraid that I don't have write access to this repo, so I cannot merge this PR.

**Pull request merged**
